test/Typesense.Tests/TypesenseClientTests.cs

```diff
@@ -1850,6 +1850,18 @@ public async Task Can_create_snapshot()
             response.Success.Should().BeTrue();
         }
     }
+    
+    [Fact, TestPriority(36)]
+    public async Task Can_Compact_Disk()
```

Comment:
Change this to `Can_compact_disk` to follow the style of the naming of the test methods.